brat/Brat/Checker/Monad.hs

```diff
@@ -172,7 +172,7 @@ tlup (m, c) = req (TLup (m, c)) >>= \case
     Brat -> Kernel
     Kernel -> Brat
 
-lookupAndUse :: UserName -> KEnv
+lookupAndUse :: QualName -> KEnv
```

Comment:
Similarly, if it's something that might be used then will it really be a QualName?

Reply:
variables are parsed as qualified names, so they can include prefixes

Reply:
Even kernel names? Can declare a variable of type qubit whose name has a prefix?

Reply:
Ah I see, we could be more strict about kernel variables, you're right